### PR TITLE
Chore: adds alert banner

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksadds-research-banner-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1203467793559483) 🎟️

## 🗒️ What

Adds an alert banner for the research survey.

### CODE REVIEW

File: src/layouts/base-new/index.tsx [hunk 1]
```diff
@@ -26,6 +28,17 @@ const { ConsentManager, openConsentManager } = createConsentManager({
 	preset: 'oss',
 })
 
+// TODO find better place for this
+const shouldShowAlertBanner = true
```
Comment (kendallstrautman) [inline src/layouts/base-new/index.tsx:32]: put this here to get product / designs eyes on it. but it should live elsewhere.... `data` , `content`, or `config`? 🤔

**Review by braydenlove — APPROVED** [review]
This looks great. Can we add a period like so?

Research | Help improve navigation and content organization by answering a short survey`.` **Start Survey**

If we can update it I think we are set.

File: src/layouts/base-new/index.tsx [hunk 2]
```diff
@@ -26,6 +28,17 @@ const { ConsentManager, openConsentManager } = createConsentManager({
...
+const alertBannerData = {
+	product: 'hashicorp' as HashiCorpProduct,
+	tag: 'Research',
+	url: 'https://t.maze.co/130555132',
+	text: 'Help improve navigation and content organization by answering a short survey',
```
Comment (kendallstrautman) [inline src/layouts/base-new/index.tsx:37]:
```suggestion
	text: 'Help improve navigation and content organization by answering a short survey.',
```

File: src/layouts/base-new/index.tsx (hunk 1 above)
Comment (kendallstrautman) [reply src/layouts/base-new/index.tsx:32]: I put it in `data`

**Review by zchsh — APPROVED** [review]

**Review by braydenlove — APPROVED** [review]
Just re-approving so I can show Jasmine how to approve!

**Pull request merged**